From b0504a349edff1dd5902058f8734a507ae08b9a9 Mon Sep 17 00:00:00 2001
From: Mathew McBride <matt@traverse.com.au>
Date: Thu, 12 Feb 2026 11:16:24 +1100
Subject: [PATCH 1/3] platform: raspberry-pi: remove unused variable in
 platform_get_ssid_default

Fixes this compile error:
platform/raspberry-pi/platform_pi.c: In function 'platform_get_ssid_default':
platform/raspberry-pi/platform_pi.c:196:9: error: unused variable 'ret' [-Werror=unused-variable]
|   196 |     int ret = 0;
---
 platform/raspberry-pi/platform_pi.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/platform/raspberry-pi/platform_pi.c b/platform/raspberry-pi/platform_pi.c
index 04f721f..f96957b 100644
--- a/platform/raspberry-pi/platform_pi.c
+++ b/platform/raspberry-pi/platform_pi.c
@@ -193,8 +193,6 @@ int platform_get_keypassphrase_default(char *password, int vap_index)
 
 int platform_get_ssid_default(char *ssid, int vap_index)
 {
-    int ret = 0;
-
     wifi_hal_dbg_print("%s:%d \n", __func__, __LINE__);
     /* if the vap_index is that of mesh STA or mesh backhaul then try to obtain the ssid from
        /nvram/EasymeshCfg.json file */
-- 
2.51.2

